configure.ac

```
@@ -1631,6 +1631,9 @@ AC_ARG_ENABLE(log-daemon-helpers,
 ])
 m4_include([src/log/helpers.m4])
 
+dnl Check for libraries required by ACL external helpers
```

Typo (or unusual phrasing that does not match `--enable-external-acl-helpers` and other text):

```suggestion
dnl Check for libraries required by external ACL helpers
```